Core/include/Acts/Seeding/SeedFilter.ipp

```diff
@@ -73,14 +73,16 @@ void SeedFilter<external_spacepoint_t>::filterSeeds_2SpFixed(
               });
   }
 
+  // vector containing the radius of all compatible seeds
+  std::vector<float> compatibleSeedR;
+  compatibleSeedR.reserve(m_cfg.compatSeedLimit);
```

Comment:
what is the usual value for `m_cfg.compatSeedLimit`? Can we set a reasonable (i.e. 0) default value for it in order not to allocate too much memory (or worse having a `bad_alloc ` thrown at us)?

Reply:
the default value of `compatSeedLimit` is 2. For PPP it is 4 and for SSS it is 3

Reply:
ok with this then